src/njoy21/input/GROUPR/src/readCard9List.hpp

```diff
@@ -0,0 +1,9 @@
+template< typename Istream>
+static std::vector< Card9 > readCard9List( Istream& is ){
+  std::vector< Card9 > card9List;
+
```

Comment:
I spoke with @kahlerac this morning. He explained that this is an obscure way of saying that `mtname` can be up to 60 characters (4*15) in length. There is no limit to the number of Card9s there can be. Page 243 of the NJOY2016 manual shows an example with 7 Card9s (but no terminating Card9).